testprojects/12.gradle/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Language server somehow recognizes build.gradle and compile this project.
+# Ignore following generated files to keep SCM clean.
+.gradle/
+.settings/
+.project

```

Comment:
some test project is eclipse project itself, how to handle the case?

Reply:
Note that this file is under the test project 12, it only takes effect within the folder

Reply:
please add .idea

Reply:
I'm not going to do that in this PR. `.idea` should be added in the global .gitignore file (if needed). 
Actually I don't think we need to add that because this project is not supposed to be open in IntelliJ IDEA.